.github/workflows/deploy-pack.yml

```
@@ -0,0 +1,181 @@
+# There is no manual way to call this out to run this on tags via UI.
+# See: https://github.community/t/workflow-dispatch-from-a-tag-in-actions-tab/130561
+on: workflow_dispatch
+name: Continuous Integration - Pack
```

this still says CI; should be
```suggestion
name: Deployment - Pack
```
probably?